jenkins_pipelines/environments/manager-5.0-qe-build-validation-NUE

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env groovy
+
+node('sumaform-cucumber') {
+    properties([
+        buildDiscarder(logRotator(numToKeepStr: '10', daysToKeepStr: '30', artifactNumToKeepStr: '10')),
+        disableConcurrentBuilds(),
+        pipelineTriggers([cron('H H/4 * * *')]),
+        parameters([
+            string(name: 'cucumber_gitrepo', defaultValue: 'https://github.com/uyuni-project/uyuni.git', description: 'Testsuite Git Repository'),
```

Comment:
For me those parameters are not the build validation parameters.
You are missing arguments  like must_run_core / must_sync / must_add_MU_repositories / ...

Reply:
Fixed.